src/server/socket.js

```diff
@@ -309,6 +309,7 @@ SocketServer.prototype._subscribeTo = function (host, msg, handler, once) {
     var socket = this._hostSockets[host],
         method = once ? 'once' : 'on';
     if (socket) {
+        socket.removeListener(msg, handler);
```

Comment:
I wonder if this should be a more targeted fix rather than doing this for every `_subscribeTo` call. In general, this method should support multiple subscribers to the same message. 

@busykai - what do you think?